src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -57,7 +85,7 @@
         [quo/wallet-graph {:time-frame :empty}]]
        [rn/flat-list
         {:style      style/accounts-list
-         :data       account-cards
+         :data       (refactor-data)
```

Comment:
what is the name `refactor-data` ? why it's named refactor?

Reply:
It's refactoring the input data and prepare a well-formed data for the render-function